INSTALL

```diff
@@ -29,6 +29,9 @@ Basic Installation
 install this package:
    `./bootstrap && ./configure && make && make install`
 
+   For Ubuntu 16.04 it is recommended to do the following before building:
+$ sudo apt install git automake curl gcc libncurses-dev libreadline-dev libssl-dev libsystemd-dev libtool make wget
```

Comment:
It's actually either `curl` or `wget`, not both.
Also, it would be better to put the prerequisite info above the installation instruction.

Reply:
> It's actually either `curl` or `wget`, not both.

Okay. I thought it didn't hurt to have both 😊

> Also, it would be better to put the prerequisite info above the installation instruction.

Will do!

Thanks.